prisma/schema.prisma

```diff
@@ -85,9 +85,9 @@ model HealthcareProfessional {
   HealthcareProfessionalDegree    HealthcareProfessionalDegree[]
   spokenLanguages                 HealthcareProfessionalSpokenLanguage[]
 
-  contact Contact @relation(fields: [contactId], references: [id])
+  contact Contact? @relation(fields: [contactId], references: [id])
```

Comment:
As I've worked with the data I am lightening up some of the requirements to make it easier to use